modules/axhal/Cargo.toml

```diff
@@ -55,7 +55,7 @@ aarch64-cpu = "9.4"
 tock-registers = "0.8"
 arm_gicv2 = "0.1"
 arm_pl011 = "0.1"
-arm_pl031 = { version = "0.1", optional = true }
+arm_pl031 = { version = "0.2", optional = true }
```

Comment:
as well as percpu crate

percpu = "0.1" => percpu = "0.1.3"

Reply:
It already pinned in `Cargo.lock`

Reply:
get it.
lgtm